### DISCUSSION TITLE
Use `Sudo` command and this happend

### DISCUSSION THREAD

**Discussion by nobir** [vivek9patel/vivek9patel.github.io#110] [General]
## Try the `sudo` command to get this picture 😂. Man! It got me there.

![used-sudo-command](https://github.com/user-attachments/assets/6ec99f0e-5d33-4a2f-874a-feac7fa5f5c4)

**Comment by ZENGHAOCHEN5122** [comment]
### Yes,that's right.And I run `echo Ihateyou`...😂🤣Laughing out loud!
-------
<img width="1184" height="753" alt="image" src="https://github.com/user-attachments/assets/df378f37-f1dc-4c17-8172-bdf8c9ab26ff" />

**Reply by ZENGHAOCHEN5122** [reply]
_**### Try that `sudo`!That's fun!Ha ha ha!😄!**_
